dist/pythonlibs/riotctrl_shell/gnrc.py

```diff
@@ -25,6 +25,8 @@ def __init__(self):
                                   r"( rssi=(?P<rssi>-?\d+) dBm)?"
                                   r"( time=(?P<rtt>\d+.\d+) ms)?"
                                   r"(?P<dup> \(DUP\))?")
+                                  r"(response truncated by (?P<truncated>\d+) byte"
```

Comment:
You forgot to move the parenthesis at the end of line above to the last line of your addition. That's what `flake8` is complaining about ;-).

Reply:
Would be great to have a test case for this in `vim dist/pythonlibs/riotctrl_shell/tests/test_netif.py`. This way you would have caught the error maybe earlier :-).

Reply:
should be fixed now

Reply:
The test case is still missing and `tools-test` is still complaining ...